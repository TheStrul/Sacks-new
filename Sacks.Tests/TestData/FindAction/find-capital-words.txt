M KORS Wonderlust Wom EDP (100ml+SG 100ml+BL 100ml+TS10ml)
MOSCHINO Toy Boy EDP (100ml+BG 100ml+ 10ml)
MOSCHINO Toy2Pearl EDP (100+SG100+BL100+10)
NARCISO RODRIGUEZ For Her EDT (100ml+BL 50ml+PS10ml)
VERSACE Dylan Blue Men EDT (100ml+BSG 150ml+EDT 5ml)
VERSACE Dylan Blue Pour Femme Wom EDP (100ml+BS100ml+BL100ml+5ml)
VERSACE Dylan Purple Wom EDP (100ml+SG 100ml+BL100ml+5ml)
VERSACE Eros Energy EDP (100ml+BSG 150ml+ 5ml)
VERSACE Eros Flame Men EDP (100ml+BSG 150ml+EDT 5ml)
VERSACE Eros Pour Femme EDP (100ml+BSG 100ml+BL100ml+ EDP5ml)
VERSACE Pour Homme EDT (100ml+BSG 150ml+ 5ml)
VERSACE Yellow Diamond Wom EDT (90ml+BL 100ml+BSG 100ml+ 5ml)
D&G Light Blue Intense Wom EDP TST (100ml)
D&G Pour Homme EDT TST (125ml)
D&G The One Gold Intense Wom EDP TST (75ml)
D&G The One Men Intense EDP TST (100ml)
MOSCHINO Cheap & Chic Wom EDT TST (100ml)
MOSCHINO Toy Boy Men EDP TST (100ml)
MOSCHINO Toy2 Bubble Gum EDT TST (100ml)
MOSCHINO Toy2 Wom TST EDP (100ml)
Acme UPTOWN BRAND Co