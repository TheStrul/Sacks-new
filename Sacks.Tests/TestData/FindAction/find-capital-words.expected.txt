M KORS Wonderlust Wom EDP (100ml+SG 100ml+BL 100ml+TS10ml) | M KORS
MOSCHINO Toy Boy EDP (100ml+BG 100ml+ 10ml) |  MOSCHINO
MOSCHINO Toy2Pearl EDP (100+SG100+BL100+10) | MOSCHINO
NARCISO RODRIGUEZ For Her EDT (100ml+BL 50ml+PS10ml) | NARCISO RODRIGUEZ
VERSACE Dylan Blue Men EDT (100ml+BSG 150ml+EDT 5ml) | VERSACE
VERSACE Dylan Blue Pour Femme Wom EDP (100ml+BS100ml+BL100ml+5ml) | VERSACE
VERSACE Dylan Purple Wom EDP (100ml+SG 100ml+BL100ml+5ml) | VERSACE
VERSACE Eros Energy EDP (100ml+BSG 150ml+ 5ml) | VERSACE
VERSACE Eros Flame Men EDP (100ml+BSG 150ml+EDT 5ml) | VERSACE
VERSACE Eros Pour Femme EDP (100ml+BSG 100ml+BL100ml+ EDP5ml) | VERSACE
VERSACE Pour Homme EDT (100ml+BSG 150ml+ 5ml) | VERSACE
VERSACE Yellow Diamond Wom EDT (90ml+BL 100ml+BSG 100ml+ 5ml) | VERSACE
D&G Light Blue Intense Wom EDP TST (100ml) | D&G
D&G Pour Homme EDT TST (125ml) | D&G
D&G The One Gold Intense Wom EDP TST (75ml) | D&G
D&G The One Men Intense EDP TST (100ml) | D&G
MOSCHINO Cheap & Chic Wom EDT TST (100ml) | MOSCHINO
MOSCHINO Toy Boy Men EDP TST (100ml) | MOSCHINO
MOSCHINO Toy2 Bubble Gum EDT TST (100ml) | MOSCHINO
MOSCHINO Toy2 Wom TST EDP (100ml) | MOSCHINO
Acme UPTOWN BRAND Co | ""